scenarios/setup.xml

```diff
@@ -54,36 +54,36 @@
-        <step name="Up4-App-Uninstall" requires="Onos-Wait-For-Ready"
-              exec="onos-app ${OCI} uninstall org.omecproject.up4"/>
-        <step name="Up4-App-Wait-For-Uninstalled" requires="^"
-              exec="docker-log-grep ${ODI} 'Application org.omecproject.up4 has been uninstalled'"/>
-        <step name="Up4-Oar-Exists" requires="^"
-              exec="test -f ${UP4_ROOT}/app/app/target/up4-app-1.2.0-SNAPSHOT.oar"/>
-        <step name="Up4-App-Install-Local-Build" requires="^" delay="5"
-              exec="onos-app ${OCI} reinstall! ${UP4_ROOT}/app/app/target/up4-app-1.2.0-SNAPSHOT.oar"/>
-
-
-        <parallel var="${OD#}">
-            <!-- FIXME: ${OC#} pointing to the same instance localhost, should we use a container that can resolve onos1/2/3 hostnames? -->
-            <step name="Onos-Check-Up4-App-${#}"
-                  exec="onos-check-apps ${OC#} org.omecproject.up4 includes"
-                  requires="Up4-App-Install-Local-Build"/>
-            <step name="Up4-Set-Log-Debug-${#}"
-                  exec="onos ${OC#} log:set DEBUG org.omecproject.up4"
-                  requires="Onos-Check-Up4-App-${#}"/>
-            <step name="Onos-Enable-Grpc-Log-${#}"
-                  exec="onos ${OC#} cfg set org.onosproject.grpc.ctl.GrpcChannelControllerImpl enableMessageLog true"
-                  requires="Onos-Check-Up4-App-${#}"/>
-            <step name="Onos-Check-Components-${#}" exec="onos-check-components ${OC#}"
-                  delay="5" requires="Onos-Check-Up4-App-${#}"/>
-            <!-- FIXME: re-enable log checking after bundle refresh will be fixed in ONOS 2.5
-                 Bundle refresh causes many exception that are deemed non harmful. Since we also
-                 check component status in the previous steps, we have enough confidence that the
-                 system is a in a good state at this point...
-            <step name="Onos-Check-Logs-${#}" exec="onos-check-logs ${OD#}"
-                  requires="Onos-Check-Components-${#}"/>
-            -->
-        </parallel>
+        <group name="Reinstall-Up4" if="${PUSH_UP4}" requires="Onos-Wait-For-Ready">
+            <step name="Up4-App-Uninstall" requires="Onos-Wait-For-Ready"
+                  exec="onos-app ${OCI} uninstall org.omecproject.up4"/>
+            <step name="Up4-App-Wait-For-Uninstalled" requires="^"
+                  exec="docker-log-grep ${ODI} 'Application org.omecproject.up4 has been uninstalled'"/>
+            <step name="Up4-Oar-Exists" requires="^"
+                  exec="test -f ${UP4_ROOT}/app/app/target/up4-app-1.2.0-SNAPSHOT.oar"/>
+            <step name="Up4-App-Install-Local-Build" requires="^" delay="5"
+                  exec="onos-app ${OCI} reinstall! ${UP4_ROOT}/app/app/target/up4-app-1.2.0-SNAPSHOT.oar"/>
+            <parallel var="${OD#}">
+                <!-- FIXME: ${OC#} pointing to the same instance localhost, should we use a container that can resolve onos1/2/3 hostnames? -->
```

Comment:
we can use the apache client and remove this fixme

Reply:
I plan to fix this on a separate PR

Reply:
See: https://github.com/omec-project/up4/pull/274